test/test.js

```diff
@@ -18,557 +19,653 @@ var describeStatusMessage = !/statusMessage/.test(http.IncomingMessage.toString(
   ? describe.skip
   : describe
 
-describe('finalhandler(req, res)', function () {
-  describe('headers', function () {
-    it('should ignore err.headers without status code', function (done) {
-      request(createServer(createError('oops!', {
-        headers: { 'X-Custom-Header': 'foo' }
-      })))
-        .get('/')
-        .expect(shouldNotHaveHeader('X-Custom-Header'))
-        .expect(500, done)
-    })
-
-    it('should ignore err.headers with invalid res.status', function (done) {
-      request(createServer(createError('oops!', {
-        headers: { 'X-Custom-Header': 'foo' },
-        status: 601
-      })))
-        .get('/')
-        .expect(shouldNotHaveHeader('X-Custom-Header'))
-        .expect(500, done)
-    })
+const servers = [
+  ['http', createHTTPServer],
+  ['http2', createHTTP2Server]
+]
+
+for (var i = 0; i < servers.length; i++) {
+  const name = servers[i][0]
+  const createServer = servers[i][1]
```

Comment:
This package supports older node versions. I am nearly positive this will end up failing CI because of it. We will be dropping those in the next major, but ideally we can land this in the current major first since that is what webpack will be relying on.

Reply:
It should work fine now, everything went well locally on `0.8`